setup.cfg

```diff
@@ -1,3 +1,56 @@
+[metadata]
+name = Django
+version = attr: django.__version__
+url = https://www.djangoproject.com/
+author = Django Software Foundation
+author_email = foundation@djangoproject.com
+description = A high-level Python Web framework that encourages rapid development and clean, pragmatic design.
+long_description = file: README.rst
+license = BSD
```

Comment:
I know that this was taken over from what is currently there in ``setup.py``, I  think we should be more clear.

Strictly speaking, Python is licensed under [BSD 3-Clause "New" or "Revised" License](https://spdx.org/licenses/BSD-3-Clause.html) which isn't clear from this and is not possible to specify in the classifiers. We should either use the full name or, better yet, the SPDX identifier: `BSD-3-Clause`. [1]

In addition, Django contains a copy of [Python License 2.0](https://spdx.org/licenses/Python-2.0.html) which was added as in the past some code was copied from the Python Standard Library. Strictly speaking that means that Django is (or should be) distributed under the terms of both licenses. That is also not clear here. Looking back through the history of `LICENSE.python` it was originally added for a [backport of `weakref`](https://github.com/django/django/commit/7959aff8be437f6cc9cbab2cdae86a6aad24f925) which has since been removed. I thought that we could potentially remove it from master again, but grepping for the license filename, we have custom version of `urllib.parse.parse_qsl()` as [`django.utils.http.limited_parse_qsl()`](https://github.com/django/django/commit/929684d6ee0efb5afad51dc584489d0437d2451b). [2]

So it seems to me that perhaps we ought to make this `license = (BSD-3-Clause AND Python-2.0)` and add `License :: OSI Approved :: Python Software Foundation License` to `classifiers`.

@carltongibson @felixxm I think this needs some looking into what the correct course of action is.

[1] Note that there is an ongoing [discussion](https://discuss.python.org/t/improving-license-clarity-with-better-package-metadata/2154) on how to make this better in the (probably now, not too far) future, but I think we can do better in the short term.
[2] I see that Python 3.8 has a `max_num_fields` argument that mirrors the `fields_limit` argument in the custom version - am prepping a PR to handle updating to a backport from 3.8 instead of the current implementation so that it can eventually be removed.

Reply:
> PR to handle updating to a backport from 3.8

See #12017.

Reply:
I'll happily adjust the license lines to the most correct and explicit. I'll await instruction from @carltongibson @felixxm on how to proceed.